leo/core/leoQt.py

```diff
@@ -38,43 +39,50 @@
             else:
                 fail = True
 # Complete the imports.
+
+QString = g.u
+QtConst = QtCore = QtGui = QtWidgets = QUrl = None
+QtDeclarative = Qsci = QtSvg = QtWebKit = QtWebKitWidgets = None
+phonon = uic = None
+qt_version = '<no version>'
+printsupport = None
+
 if fail:
     isQt5 = False
-    QString = g.u
-    Qt = QtConst = QtCore = QtGui = QtWidgets = QUrl = None
-    QtDeclarative = Qsci = QtSvg = QtWebKit = QtWebKitWidgets = None
-    phonon = uic = None
-    qt_version = '<no version>'
-    printsupport = None
+    Qt = None
 elif isQt5:
     try:
         from PyQt5 import QtCore
         from PyQt5 import QtGui
         from PyQt5 import QtWidgets
         from PyQt5.QtCore import QUrl
-        QtConst = QtCore.Qt
         printsupport = Qt
     except ImportError:
         print('leoQt.py: can not fully import PyQt5.')
+        print(traceback.format_exc())
+        print("")
```

Comment:
Having this in the code would have been useful for https://groups.google.com/forum/#!topic/leo-editor/ghiIN7irzY0 and https://groups.google.com/forum/#!topic/leo-editor/xpLE-LHdYrk

If having the full stack trace displayed is too much, at the very least, the exception message would be helpful. Without this extra message, the user needs to resort to one of the following to get more information:

1. Manually run the imports from a python interpreter in order to see the message
2. Use pdb to step through this code in the debugger
3. Change the code themselves to display more information

So it would be useful to have some form of this pull request merged